scripts/ci_test_interface.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+### Test slither-interface
+
+DIR_TESTS="tests/tools/interface" 
+
+solc-select use 0.8.19 --always-install
+
+#Test 1 - Etherscan target
+slither-interface WETH9 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
+DIFF=$(diff crytic-export/interfaces/IWETH9.sol "$DIR_TESTS/test_1.sol")
+if [  "$DIFF" != "" ]
+then
+    echo "slither-interface test 1 failed"
+    cat test_1.sol
```

Comment:
Should this be `crytic-export/interfaces/IWETH9.sol`? ditto for the other occurrences below.